editor.planx.uk/src/pages/FlowEditor/components/Sidebar/index.tsx

```diff
@@ -167,102 +151,114 @@ const Sidebar: React.FC = React.memo(() => {
   };
 
   return (
-    <Root isMinimised={isSidebarMinimised}>
-      <SidebarWrapper>
-        <StyledToggleButton onClick={togglePreview} value="toggleSidebar">
-          {isSidebarMinimised ? <ChevronLeftIcon /> : <ChevronRightIcon />}
-        </StyledToggleButton>
-        <Header>
-          <Box width="100%" display="flex">
-            <input type="text" disabled value={urls.preview} />
-
-            <Permission.IsPlatformAdmin>
-              <Tooltip arrow title="Open draft service">
-                <Link
-                  href={urls.draft}
-                  target="_blank"
-                  rel="noopener noreferrer"
-                  color="inherit"
-                >
-                  <OpenInNewOffIcon />
-                </Link>
-              </Tooltip>
-            </Permission.IsPlatformAdmin>
+    <Root>
+      <Collapse 
+        in={showSidebar} 
+        orientation="horizontal" 
+        collapsedSize={SIDEBAR_WIDTH_MINIMISED} 
+        sx={{ height: "100%" }}
+        easing={"ease-in-out"}
+        timeout={200}
+      >
```

Comment:
Diff here is a bit weird - I've just wrapped this component in a MUI Collapse ([docs](https://mui.com/material-ui/api/collapse/)) instead of controlling this via variables passed to a styled component.